spec/requests/home_request_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe "Homes", type: :request do
```

Comment:
```suggestion
describe "Homes" do
```
sure the generators added all the other bits but we dont need them so clean it up imo